android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.reactnativeshopify">
```

Comment:
Thanks for reminding me to check the application name and its use. It's used for marketing attribution and appended to each checkout object. Shopify uses it for internal analytics. 

On iOS, Shopify takes it from the _Info.plist_ file:

```
- (NSString *)applicationName
{
	return [[self appBundle] objectForInfoDictionaryKey:(id)kCFBundleNameKey] ?: @"";
}
```
It's then used in the checkout:

`checkout.marketingAttribution = @{@"medium": @"iOS", @"source": self.applicationName};`

We're using a variable _$PRODUCT_NAME_ in Shoutem's application for iOS. How do we handle this with Android? 

I think that the bridge's AndroidManifest.xml can stay as it is, but we do need a way to set the application name from the application side. Did we already have a similar case?